.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# WKT to check files match with the released artifact
+/modules/sync/protocolbuffers/wellknowntypes/state.json @jhump @pkwarren @unmultimedio
```

Comment:
Do we need to do anything else to enable this, like a module setting to actually require approvals from code owners? (I haven't messed around with code owners in quite some time...)

Reply:
Imo with the fact that we're notified that's enough, usually people that approve PRs from this repo are no more than 4-5 people. But if we want to lock it down to _require_ a CODEOWNER approval (if it's not already on), that'd be @rubensf

Reply:
Just enabled the requirement of code owners.

Note that even if the author is a code owner, one of the reviewers will need to be a _different_ code owner.